dotcom-rendering/src/web/components/Card/components/LI.tsx

```diff
@@ -1,28 +1,32 @@
 import { css } from '@emotion/react';
-import { from, until } from '@guardian/source-foundations';
+import { from, space, until } from '@guardian/source-foundations';
 import { verticalDivider } from '../../../lib/verticalDivider';
 import { verticalDividerWithBottomOffset } from '../../../lib/verticalDividerWithBottomOffset';
 
+/**
+ * This value needs to match the one set
+ * in the `verticalDividerWithBottomOffset`
+ */
```

Comment:
Which value of `verticalDividerWithBottomOffset` this needs to match and how is this accomplished? Sorry, this might be a silly question but I'm not very familiar with this and not sure I'm following here.

Reply:
It’s the one in [`lib/verticalDividerWithBottomOffset`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/lib/verticalDividerWithBottomOffset.ts) imported just above.